docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+	Powertools will handle the request from the Agent, parse, validate, and route it to the correct method in your code.
+	The response is then validated and formatted back to the Agent.
+
+	<center>
+	```mermaid
+	--8<-- "docs/core/event_handler/bedrock_agents_getting_started.mermaid"
+	```
+	</center>
+
+### Validating input and output
+
+You can define the expected format for incoming data and responses by using type annotations.
+Define constraints using standard Python types, [dataclasses](https://docs.python.org/3/library/dataclasses.html) or [Pydantic models](https://docs.pydantic.dev/latest/concepts/models/).
+Pydantic is a popular library for data validation using Python type annotations.
+
+=== "Lambda handler"
+	This example uses [Pydantic's EmailStr](https://docs.pydantic.dev/2.0/usage/types/string_types/#emailstr){target="_blank"} to validate the email address passed to the `schedule_meeting` function.
+    The function then returns a boolean indicating if the meeting was successfully scheduled.
+
+	```python hl_lines="1 2 16-18"
+	--8<-- "examples/event_handler_bedrock_agents/src/getting_started_with_validation.py"
+	```
+
+	1. No need to add the `enable_validation` parameter, as it's enabled by default.
+	2. Describe each input using human-readable descriptions
+	3. Add the typing annotations to your parameters and return types, and let the event handler take care of the rest
+
+=== "OpenAPI schema"
+
+	```json
+	--8<-- "examples/event_handler_bedrock_agents/src/getting_started_with_validation_schema.json"
+	```
+
+=== "Input payload"
+
+	```json hl_lines="6-13 20"
+	--8<-- "examples/event_handler_bedrock_agents/src/getting_started_with_validation.json"
+	```
+
+=== "Output payload"
+
+	```json hl_lines="10"
+	--8<-- "examples/event_handler_bedrock_agents/src/getting_started_with_validation_output.json"
+	```
+
+#### When validation fails
+
+If the request validation fails, your event handler will not be called, and an error message is returned to Bedrock.
+Similarly, if the response fails validation, your handler will abort the response.
+
+???+ info "What does this mean for my Agent?"
+	The event handler will always return a response according to the OpenAPI schema.
+	A validation failure always results in a [422 response](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422).
+	However, how Amazon Bedrock interprets that failure is non-deterministic, since it depends on the characteristics of the LLM being used.
+
+=== "Input payload"
+
+	```json hl_lines="11"
+	--8<-- "examples/event_handler_bedrock_agents/src/validation_failure_input.json"
+	```
+
+=== "Output payload"
+
+	```json hl_lines="10"
+	--8<-- "examples/event_handler_bedrock_agents/src/validation_failure_output.json"
+	```
+
+<center>
+```mermaid
+--8<-- "docs/core/event_handler/bedrock_agents_validation_sequence_diagram.mermaid"
+```
+</center>
+
+### Generating OpenAPI schemas
+
+Use the `get_openapi_json_schema` function provided by the resolver.
```

Comment:
The first part of the line doesn't have explanation of what you're using this function call for, i.e.:
> Use the `get_openapi_json_schema` function provided by the resolver to **blow up the death star**